protocol/query/method/message/data/dataCashTransaction.json

```diff
@@ -9,6 +9,11 @@
     },
     "action": {
       "const": "post"
+    },
+	"transaction_id": {
+        "type": "string",
+        "contentEncoding": "base64",
+        "description": "SHA 256 Hash over base64 of transaction - must be verified"
```

Comment:
Let's add "must be verified" to protocol spec by saying that this value is untrusted and used for consensus reasons.

Reply:
I'm not sure if I understand well your comment. You mean that when I write protocol spec, I should specify that the transaction_id must be verified because the transaction is untrusted and it can be used for consensus? Or did I misunderstood ?

Reply:
`transaction_id` must be verified by hashing the transaction value (a base64encoded string). One cannot trust that the transaction_id field is the actual hash of the transaction value.

Reply:
You mean in this file: https://github.com/dedis/popstellar/blob/master/docs/messageData.md  ?
(We still didn't write anything inside of it. I going to do it during the following week or the eastern holidays)

Reply:
Yes perfect.